apps/notification_bot/utils.ts

```diff
@@ -1,8 +1,95 @@
-export const getMessage = (link, userName, users) => (`
+import { Octokit } from '@octokit/core'
+const octokit = new Octokit({
+    auth: process.env.NOTIFICATION_BOT_CONFIG && JSON.parse(process.env.NOTIFICATION_BOT_CONFIG)?.github_auth_key
+})
+const AVAILABLE_TASK_TYPES = ['SBERDOMA']
+
+export const getPullRequestMessage = (link, userName, users) => (`
 ******************************
 Pull request opened.
 Author: ${userName}.
 ${users.map(([userName]) => `@${userName}`).join(', ')}
 Link: ${link}.
 ******************************
-`)
\ No newline at end of file
+`)
+
+export const getFormattedCommits = (commits) => {
+    const formattedCommit = commits
+        .map(({ message }) => {
+            const [task_number] = message.split(' ')
+            return task_number
+        })
+        .filter((task_number) => {
+            const [taskType] = task_number.split('-')
+
+            return AVAILABLE_TASK_TYPES.includes(taskType)
+        })
+
+    return Array.from(new Set<string>(formattedCommit))
+        .sort((leftCommit, rightCommit) => {
+            const [, leftCommitNumber] = leftCommit.split('-')
+            const [, rightCommitNumber] = rightCommit.split('-')
+
+            return Number(leftCommitNumber) > Number(rightCommitNumber) ? 1 : -1
+        }).map((commit) => {
+            return `https://doma.atlassian.net/browse/${commit}`
```

Comment:
May be move to constant

Reply:
i'm not sure we need it here

Reply:
I think, better place for configuration is `config` node in `package.json`